test/integration/preflight/check-yum-update/test-upgrade-dependency-missing.yml

```diff
@@ -0,0 +1,10 @@
+- hosts: localhost
+  tasks:
+
+  - include: tasks/create-test-container.yml
+
+  - name: test should fail; cannot yum update due to dependencies
+    delegate_to: dockertest
+    action: check_yum_update
```

Comment:
I think we start a preflight playbook. For tasks that look more complex than:

    - action: check_yum_update

... we can use includes to ensure the integration tests run exactly the same thing.